compiler/AST/ImportStmt.cpp

```diff
@@ -29,6 +30,25 @@
 ************************************** | *************************************/
 
 ImportStmt::ImportStmt(BaseAST* source) : VisibilityStmt(E_ImportStmt) {
+  this->modRename = "";
```

Comment:
I don't understand why use `""` rather than `NULL` here. But if you continue with `""` can you make it `astr("")` (and likewise in UseStmt)? I just think all these strings should be astrs. The uses of it can remain the same.